scripts/devenv-builder/configure-vm.sh

```diff
@@ -52,23 +52,24 @@ fi
 echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee /etc/sudoers.d/microshift
 sudo dnf clean all -y
 sudo dnf update -y
-sudo dnf install -y git cockpit make golang jq selinux-policy-devel rpm-build jq bash-completion
-sudo systemctl enable --now cockpit.socket
-
-# Install go1.19
-# This is installed into different location (/usr/local/bin/go) from dnf installed Go (/usr/bin/go) so it doesn't conflict
-# /usr/local/bin is before /usr/bin in $PATH so newer one is picked up
-GO_VER=1.19.4
-GO_ARCH=$([ "$(uname -i)" == "x86_64" ] && echo "amd64" || echo "arm64")
-curl -L -o "go${GO_VER}.linux-${GO_ARCH}.tar.gz" "https://go.dev/dl/go${GO_VER}.linux-${GO_ARCH}.tar.gz" &&
-    sudo rm -rf "/usr/local/go${GO_VER}" && \
-    sudo mkdir -p "/usr/local/go${GO_VER}" && \
-    sudo tar -C "/usr/local/go${GO_VER}" -xzf "go${GO_VER}.linux-${GO_ARCH}.tar.gz" --strip-components 1 && \
-    sudo rm -rfv /usr/local/bin/{go,gofmt} && \
-    sudo ln --symbolic /usr/local/go${GO_VER}/bin/{go,gofmt} /usr/local/bin/ && \
-    rm -rfv "go${GO_VER}.linux-${GO_ARCH}.tar.gz"
 
 if [ $BUILD_AND_INSTALL = true ] ; then
+    sudo dnf install -y git cockpit make golang jq selinux-policy-devel rpm-build bash-completion
+    sudo systemctl enable --now cockpit.socket
+    
+    # Install go1.19
+    # This is installed into different location (/usr/local/bin/go) from dnf installed Go (/usr/bin/go) so it doesn't conflict
+    # /usr/local/bin is before /usr/bin in $PATH so newer one is picked up
+    GO_VER=1.19.4
+    GO_ARCH=$([ "$(uname -i)" == "x86_64" ] && echo "amd64" || echo "arm64")
+    curl -L -o "go${GO_VER}.linux-${GO_ARCH}.tar.gz" "https://go.dev/dl/go${GO_VER}.linux-${GO_ARCH}.tar.gz" &&
+        sudo rm -rf "/usr/local/go${GO_VER}" && \
+        sudo mkdir -p "/usr/local/go${GO_VER}" && \
+        sudo tar -C "/usr/local/go${GO_VER}" -xzf "go${GO_VER}.linux-${GO_ARCH}.tar.gz" --strip-components 1 && \
+        sudo rm -rfv /usr/local/bin/{go,gofmt} && \
+        sudo ln --symbolic /usr/local/go${GO_VER}/bin/{go,gofmt} /usr/local/bin/ && \
+        rm -rfv "go${GO_VER}.linux-${GO_ARCH}.tar.gz"
+
```

Comment:
This is a good catch! Strictly speaking, we do not need these utilities unless we build the binaries.
@jogeo - can you verify that this change does not break the QE pipeline?

Reply:
@pmtk , let's wait for QE confirmation before we merge this PR.
It's LGTM for me

Reply:
@pmtk , as I suspected, this change breaks QE flow. Can we revert it to merge the other fixes for now?